pkg/destroy/gcp/targetpool.go

```diff
@@ -16,6 +18,21 @@ const (
 
 type targetPoolFilterFunc func(pool *compute.TargetPool) bool
 
+func (o *ClusterUninstaller) deleteTargetPoolByName(ctx context.Context, resourceName string) error {
+	items, err := o.listTargetPoolsWithFilter(ctx, "items(name),nextPageToken", func(pool *compute.TargetPool) bool {
+		return strings.Contains(pool.Name, resourceName)
```

Comment:
Other places (e.g. `deleteSubnetworkByName` in `pkg/destroy/gcp/subnetwork.go`) use exact comparison `==`.

Should we make it consistent? Or is there a reason to use `.Contains` here?